<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWindow</class>
 <widget class="QMainWindow" name="MyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labe1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>47</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>본인</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>47</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>상대</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>결과 확인</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>25</number>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>47</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>상대나이</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textBox1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Choi</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textBox2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Park</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>218</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFILE">
    <property name="title">
     <string>FILE</string>
    </property>
    <addaction name="checkAction"/>
    <addaction name="byeAction"/>
   </widget>
   <addaction name="menuFILE"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="checkAction">
   <property name="text">
    <string>Check Result</string>
   </property>
  </action>
  <action name="byeAction">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn2</sender>
   <signal>clicked()</signal>
   <receiver>MyWindow</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn1</sender>
   <signal>clicked()</signal>
   <receiver>MyWindow</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkAction</sender>
   <signal>triggered()</signal>
   <receiver>MyWindow</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>byeAction</sender>
   <signal>triggered()</signal>
   <receiver>MyWindow</receiver>
   <slot>bye()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clear()</slot>
  <slot>check()</slot>
  <slot>bye()</slot>
 </slots>
</ui>
